index.html

```diff
@@ -138,6 +138,18 @@ <h2>Demos</h2>
 				</div>
 
 				<div class="gallery">
+					<article class="gallery-item 3rd" itemscope="" itemtype="http://schema.org/CreativeWork">
+						<a title="Brilliant man" class="gallery-item-link" href="http://iprojects.space/bronchoscopy/index.html" target="blank" itemprop="url">
+							<img class="gallery-item-img" alt="Bronchoscopy Simulation" src="http://iprojects.space/bronchoscopy/bjs_img.jpg" itemprop="image">
```

Comment:
we need the image to be saved in /screenshots folder for perf reasons

Reply:
![bjs_002](https://user-images.githubusercontent.com/2744715/53275925-c4436680-36db-11e9-8d8b-841a15df8654.jpg)

Reply:
How I can upload to /screenshots ?